EIPS/eip-7620.md

```diff
@@ -74,7 +74,7 @@ Details on each instruction follow in the next sections.
 - caller's memory slice `[input_offset:input_size]` is used as calldata
 - execute the container and deduct gas for execution. The 63/64th rule from [EIP-150](./eip-150.md) applies.
 - increment `sender` account's nonce
-- calculate `new_address` as `keccak256(0xff || sender || salt || keccak256(initcontainer))[12:]`
+- calculate `new_address` as `keccak256(0xff || sender || salt)[12:]`
```

Comment:
"deduct `GAS_KECCAK256_WORD..." step should also be removed, along with `initcontainer_size` definition and `GAS_KECCAK...` constant